Sources/StreamChatUI/Utils/VideoPreviewLoader.swift

```diff
@@ -0,0 +1,62 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import AVKit
+import StreamChat
+import UIKit
+
+public protocol VideoPreviewLoader: AnyObject {
+    func loadPreviewForVideo(at url: URL, completion: @escaping (Result<UIImage, Error>) -> Void)
+}
+
+final class DefaultVideoPreviewLoader: VideoPreviewLoader {
+    @Atomic private var cache = Cache<URL, UIImage>()
```

Comment:
We don't need @Atomic on cache. `NSCache` is thread-safe out of the box.

Reply:
✅ Fixed